src/App.js

```diff
@@ -39,32 +68,65 @@ function App() {
     []
   );
   return (
-    <div className="App">
-      <button disabled={state.show === "all"} onClick={changeView("all")}>
+    <div
+      className="App"
+      css={{
+        background: 'linear-gradient(#e6646533, #9198e588)',
+        height: '100%'
+      }}
+    >
+      <Select disabled={state.show === 'all'} onClick={changeView('all')}>
         All
-      </button>
-      <button disabled={state.show === "done"} onClick={changeView("done")}>
+      </Select>
+      <Select disabled={state.show === 'done'} onClick={changeView('done')}>
         Done
-      </button>
-      <button disabled={state.show === "undone"} onClick={changeView("undone")}>
+      </Select>
+      <Select disabled={state.show === 'undone'} onClick={changeView('undone')}>
         Undone
+      </Select>
+      <br/>
+      <div>
+        {state.tasks
+          .filter(x =>
+            state.show === 'all' ? true : state.show === 'done' ? x.done : !x.done
+          )
+          .map(x => (
+            <TodoItem
+              id={x.id}
+              name={x.name}
+              remove={remove(x.id)}
+              check={check(x.id)}
+              done={x.done}
+            />
+          ))}
+      </div>
+      <input
+        type="text"
+        onChange={setNewTaskName}
+        value={state.newTask}
+        css={css`
+          border-width: 0;
+          padding: 5px;
+          font-size: 15px;
+        `}
+      />
+      <button
+        onClick={addNewTask}
+        css={{
+          fontFamily: 'MyFont',
+          borderWidth: 0,
+          backgroundColor: 'transparent',
+          fontSize: 16,
+          color: '#5c85c7',
+          transition: 'all 0.5s',
+          '&:hover': {
+            color: '#2db503',
+            transform: 'scale(2)'
+          }
+        }}
+      >
+        {'\uf0fe'}
```

Comment:
You should be able to use `&#61694;` (without curly braces and without single quotes) we don't need JSX to show this unicode character, BTW, it seems to be a Chinese character! 
![image](https://user-images.githubusercontent.com/1270688/73607318-cc817280-45c9-11ea-8356-6143476fbce9.png) https://www.fileformat.info/info/unicode/char/f0fe/index.htm